src/docs/signals/large-transactions/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Large Transactions
+author: Katsiaryna
+date: 2021-04-26
+description: Signal indicating transactions with big value in USD
+---
+
+## Definition
+
+**Large Transaction Signal** indicates transactions with large value in USD.
+
+In order to filter most valuable transactions for each transaction a threshold value is defined. The threshold is the USD value of the top-fifth transaction in last 30 days.
+
+If the transaction's approximate value in USD is above the threshold value, a signal is fired.
+
+Approximate transaction values are calculated by multiplying the amount of transferres coins to closest coin's price value using price data with 5 minutes interval for each transaction.
+
+### List of assets
```

Comment:
Let's make it as a heading 2 level. So on the right side will be more than just 1 nav item.
And rename to "List of supported tokens" as in Anomalies article (without ":", please)